presto-orc/src/main/java/com/facebook/presto/orc/zstd/ZstdJniCompressor.java

```diff
@@ -17,13 +17,27 @@
 import io.airlift.compress.Compressor;
 
 import java.nio.ByteBuffer;
+import java.util.OptionalInt;
 
 import static java.lang.Math.toIntExact;
+import static java.util.Objects.requireNonNull;
 
 public class ZstdJniCompressor
         implements Compressor
 {
-    private static final int COMPRESSION_LEVEL = 3; // default level
+    private static final int DEFAULT_COMPRESSION_LEVEL = 3; // default level
+    private final int compressionLevel;
+
+    public ZstdJniCompressor(OptionalInt compressionLevel)
+    {
+        requireNonNull(compressionLevel, "compressionLevel is null");
+        if (compressionLevel.isPresent()) {
```

Comment:
You can use
this. compressionLevel = compressionLevel.orElse(DEFAULT_COMPRESSION_LEVEL)